tsconfig.json

```diff
@@ -29,7 +29,7 @@
       "zlib": ["./src/alias/zlib.ts"]
     },
     "baseUrl": ".",
-    "types": ["vitest/globals", "vite/client"]
+    "types": ["vitest/globals", "vite/client", "d3-dsv"]
```

Comment:
Had to add this so that `csv-data-fetcher` could get types from d3-dsv

Reply:
Just had a look at the `package.json` and I don't see the types for d3-dsv included there. I think this also could be resolved with `yarn add -D @types/d3-dsv`. D3 doesn't ship it's own types, so we need to install them separately from the community (from DefinitelyTyped).

Reply:
I'll try this thanks!